app/graph/types/user_type.rb

```diff
@@ -104,6 +104,7 @@ def team_user(team_slug:)
       .joins(:team)
       .where("teams.slug" => team_slug, :user_id => object.id)
       .last
+    return nil if tu.nil?
     TeamUser.find_if_can(tu.id, context[:ability])
```

Comment:
May be `tu.nil? ? nil : TeamUser.find_if_can(tu.id, context[:ability])` is more readable

Reply:
Good idea, done!